### PR TITLE
[AFF] More detailed error msg

### PR DESCRIPTION
**Pull request by albertmink**

### CODE REVIEW

File: src/objects/aff/zcl_abapgit_object_common_aff.clas.abap [hunk 1]
```diff
@@ -218,12 +219,23 @@ CLASS ZCL_ABAPGIT_OBJECT_COMMON_AFF IMPLEMENTATION.
...
         LOOP AT <ls_messages> ASSIGNING <ls_message>.
+          ASSIGN COMPONENT 'MESSAGE' OF STRUCTURE <ls_message> TO <ls_msg>.
           ASSIGN COMPONENT 'TEXT' OF STRUCTURE <ls_message> TO <ls_text>.
           ASSIGN COMPONENT 'TYPE' OF STRUCTURE <ls_message> TO <ls_type>.
           ii_log->add(
               iv_msg  = <ls_text>
               iv_type = <ls_type>
               is_item = ms_item ).
+
+          CHECK <ls_msg>-msgty = 'E'.
```
Comment (larshp) [inline src/objects/aff/zcl_abapgit_object_common_aff.clas.abap:230]: https://github.com/SAP/styleguides/blob/main/clean-abap/CleanABAP.md#avoid-check-in-other-positions

prefer using IF

Comment (albertmink) [reply src/objects/aff/zcl_abapgit_object_common_aff.clas.abap:230]: agree

Comment (albertmink) [reply src/objects/aff/zcl_abapgit_object_common_aff.clas.abap]: Will replace the other CHECK statements as well

**Review by larshp — APPROVED** [review]

**Comment by larshp** [comment]
will eventually build and enable https://github.com/abaplint/abaplint/issues/2622

**Comment by albertmink** [comment]
Do you prefer to let the PR open?

**Comment by larshp** [comment]
nah, this doesnt seem dangerous

**Pull request merged**